src/gpu/intel/ocl/ocl_math_utils.h

```diff
@@ -81,6 +81,10 @@ int rnd_down(int a, unsigned int b) {
 #define MATH_UTILS_DECLARE_BF16 1
 #endif
 
+#if DST_SCALES_DT_E8M0
```

Comment:
Is there a reason we do not need to be concerned with the other buffers data types similar to the above declarations?

Reply:
e8m0 is designed for scales only. It seems that so far it is supported for dst only and there's no src or weights counterparts (I found only commented cases in benchdnn matmul_fp4 input list).
To be extended as support grows, I guess...